anime_downloader/sites/helpers/request.py

```diff
@@ -64,12 +78,12 @@ def setup_func(url: str,
                            url,
                            headers=default_headers,
                            **kwargs)
-        res.raise_for_status()
-        logger.debug(res.url)
-        # logger.debug(res.text)
-        if logger.getEffectiveLevel() == logging.DEBUG:
```

Comment:
This should probably not be removed, just moved under the if statement (`if sess != selescrape`)

Reply:
Ok, will do it tomorrow